packages/desktop-client/src/components/BankSyncStatus.tsx

```diff
@@ -59,9 +59,12 @@ export function BankSyncStatus() {
                   iconStyle={{ color: theme.pillTextSelected }}
                 />
                 <Text style={{ marginLeft: 5 }}>
-                  <Trans>
-                    Syncing... {accountsSyncingCount} account
-                    {accountsSyncingCount > 1 && 's'} remaining
+                  <Trans
+                    i18nKey="syncingAccounts"
+                    count={accountsSyncingCount}
+                    values={{ accountsSyncingCount }}
+                  >
+                    Syncing... {{ accountsSyncingCount }} account remaining
```

Comment:
Or this one 
![image](https://github.com/user-attachments/assets/6cda5551-5e4a-4efe-92f8-46b3c965d2fb)